src/index.ts

```diff
@@ -1,4 +1,16 @@
-//Uncomment to see the choose pokemon page
-//const newTemplate = document.querySelector('#choose-page-template')!;
-//const app = document.querySelector('#app')!;
-//app.innerHTML = newTemplate.innerHTML
\ No newline at end of file
+import { showAPopUpScreen } from './showAPopUpScreen';
+
+// setting up the starting page
+setTimeout(() => {
+    //Render starting page and add the help modal
+    const newTemplate = document.querySelector('#starting-page-template') as HTMLTemplateElement;
+    const app = document.querySelector('#app') as HTMLDivElement;
+    app.innerHTML = newTemplate.innerHTML;
+    const infoTemplate = document.querySelector('#info-modal-template') as HTMLTemplateElement;
+    app.innerHTML += infoTemplate.innerHTML;
+    // add event listeners
+    const howToPlayBtn = document.querySelector('#how-to-play-btn') as HTMLUListElement;
+    howToPlayBtn.addEventListener('click', () => {
+        showAPopUpScreen(document.querySelector('#info-modal-template') as HTMLDivElement, 'flex');
+    })
+}, 2000)
```

Comment:
```suggestion
const renderStartingPage = (app: HTMLDivElement) => {
    const startingPage = document.querySelector('#starting-page-template') as HTMLTemplateElement;
    app.innerHTML = startingPage.innerHTML;
    return startingPage;
}

const renderInfoModal = (app: HTMLDivElement) => {
    const infoModal = document.querySelector('#info-modal-template') as HTMLTemplateElement;
    app.innerHTML += infoModal;
    return infoModal;
}

const createHowToPlayButton = (infoModal: HTMLDivElement) => {
    const howToPlayBtn = document.querySelector('#how-to-play-btn') as HTMLUListElement;
    howToPlayBtn.addEventListener('click', () => {
        showAPopUpScreen(infoModal, 'flex');
    })
}

const renderApp = () => {
    const app = document.querySelector('#app') as HTMLDivElement;
    const startingPage = renderStartingPage(app);
    const infoModal = renderInfoModal(app)

    const howToPlayButton = createHowToPlayButton(infoModal as HTMLDivElement);
}

setTimeout(renderApp, 2000);
```

Reply:
What do you think about such approach? Do you see some pros and cons?
(Im not 100% sure it would work, because of some `TypeScript types`, but you get the idea)